packages/metal3-deploy/generated-changes/dependencies/external-dns/dependency.yaml

```diff
@@ -0,0 +1 @@
+url: packages/external-dns
```

Comment:
In #5 @jdob imported a tarball for this one, if this is because the chart is not maintained by us, we may want to use upstream url here and not copy it in our tree ?

Reply:
In this case, my best guess in interpreting the developing doc + incorporating that would be to change [packages/external-dns/package.yaml](url) to go from "local" to the upstream url, while keeping url: packages/external-dns within metal3 dependencies.

I could be entirely wrong here, the reason I say this is cause I didn't see anything in the docs about using tarballs directly.